doc/config.example.yaml

```diff
@@ -342,6 +342,14 @@ registry:
     allow_transfer:
       - 127.0.0.1
 
+    # Whitelist for AXFR zone transfers
+    #
+    # Only servers listed in this field will be allowed to initiate an AXFR zone
+    # transfer from this server. Server will not allow zone transfer to any
+    # requester if not configured. Applicable for primary and secondary servers.
+    allow_transfer:
+      - ns2.bosagora.io
```

Comment:
Can't be a domain, can it ?

Reply:
Also should be merged in the commit that implements it

Reply:
Implementing commit merged with #2825